src/pathfinder/npf/aystar.cpp

```diff
@@ -301,4 +301,9 @@ void AyStar::Init(Hash_HashProc hash, uint num_buckets)
 	 *  When that one gets full it reserves another one, till this number
 	 *  That is why it can stay this high */
 	this->openlist_queue.Init(102400);
+
+	/* Limit the number of nodes expanded by default
+	 *  This is a preventive method as this should never be left unattended
```

Comment:
What does this comment mean? The river pathfinder doesn't change it later, so it seems the comment is wrong? You could just write "Set a reasonable default limit."